### ISSUE TITLE
Setup CircleCI or similar.

### ISSUE THREAD

**Issue by sircharleswatson** [benstr/moneybot#32]
We should set up Circle soon with _**at least**_ eslint checking the code so the style is standard before merging things in to master

**Comment by benstr** [comment]
agreed, I never worked with Circle before but should not be much different than Semaphore or Codeship.

**Comment by sircharleswatson** [comment]
I've never used semaphore but I know Sam and Josh suggest Circle because it allows caching of things so it speeds up the build.

**Comment by benstr** [comment]
for sure, I want to give it a try!
